tutorial.md

```diff
@@ -9,49 +9,33 @@ In this tutorial we will go through in detail the steps required to set-up a Pyt
 
 <details><summary>Click to expand... </summary> <!-- markdownlint-disable-line MD033 -->
 
-To use the template you will need to have at least the following software tools installed,
+To use the template you will need at least the following software tools installed
 
-- [cookiecutter](https://cookiecutter.readthedocs.io/en/stable/),
-- [Git](https://git-scm.com/),
+- [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- [uv](https://docs.astral.sh/uv/getting-started/installation/)
+- [the GitHub command line interface (CLI)](https://github.com/cli/cli?tab=readme-ov-file#installation)
 
-and ideally also,
+Additionally, [pre-commit](https://pre-commit.com/) and [tox](https://tox.wiki) are required to complete some of the follow on steps for using the package generated with the template, so while they are not strictly needed, you will get more out of the tutorial if you have them installed. For the follow on exercises you will also need an [account set up on GitHub](https://github.com/join) if you don't already have one.
```

Comment:
```suggestion
Additionally, [pre-commit](https://pre-commit.com/) and [tox](https://tox.wiki) are required to complete some of the follow along steps for using the package generated with the template, so while they are not strictly needed, you will get more out of the tutorial if you have them installed. For the follow along exercises you will also need an [account set up on GitHub](https://github.com/join) if you don't already have one.
```

Reply:
["Follow on" is correct here for the sense of  "things following on from"](https://dictionary.cambridge.org/dictionary/english/follow-on) which is what I intended whenI originally wrote this in the context of running this as a tutorial at the festival. "Follow along" would I think change the meaning

Reply:
When reading it, I genuinely wasn't sure what was intended. If it's that use case then I think we should use following.